.env.example

```
@@ -5,4 +5,5 @@ DATABASE_USER = 'toto'
 DATABASE_PASS = '123password'
 SECRET_KEY = 'mySecretKey'
 PATH_FILE_CACHE = 'C:/path/redcraft/cache'
-URL_PROXY_REDCRAFT = 'http://proxy.production.redcraft.org:25580'
\ No newline at end of file
+URL_WEBHOOK_CONTACT_DISCORD = 'https://yourWebhookUrl'
```

Maybe give a bit more guidance by setting the URL by setting it to `https://discordapp.com/api/webhooks/insert_channel_id_here/insert_token_here`

```suggestion
URL_WEBHOOK_CONTACT_DISCORD = 'https://discordapp.com/api/webhooks/insert_channel_id_here/insert_token_here'
```